projects/kit/components/line-clamp/line-clamp.style.less

```diff
@@ -18,4 +18,9 @@
     -webkit-box-orient: vertical;
     overflow: hidden;
     overflow-wrap: anywhere;
+
+    &,
+    &::ng-deep > * {
```

Comment:
This should be unnecessary if we rollback last condition here:
https://github.com/taiga-family/taiga-ui/blob/main/projects/kit/components/line-clamp/line-clamp.component.ts#L137
Please see why it was added and if we can get rid of it.

Reply:
fixed